hr_attendance_modification_tracking/tests/test_hr_attendance.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 Creu Blanca
+# Copyright 2021 Landoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo.tests.common import TransactionCase
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT as DF
+
+
+class TestHrAttendanceTracking(TransactionCase):
+    def setUp(self):
+        super(TestHrAttendanceTracking, self).setUp()
+        self.hr_attendance = self.env["hr.attendance"]
+
+    def test_attendance_edit(self):
```

Comment:
I think it would be better to split the tests into some tests in order to focus on single cases than all cases in one test.